reference.go

```diff
@@ -35,8 +35,8 @@ import (
 )
 
 const (
-	// NameTotalLengthMax is the maximum total number of characters in a repository name.
-	NameTotalLengthMax = 255
+	// RepositoryNameTotalLengthMax is the maximum total number of characters in a repository name.
+	RepositoryNameTotalLengthMax = 255
```

Comment:
As this const is exported, we should probably alias the old one, and mark it deprecated;


```go
	// NameTotalLengthMax is the maximum total number of characters in a repository name.
	//
	// Deprecated: use [RepositoryNameTotalLengthMax] instead.
	NameTotalLengthMax = RepositoryNameTotalLengthMax
 ```

Reply:
Good points, thank you!